internal/cmd/local/command_kafka_start.go

```diff
@@ -82,8 +82,14 @@ func (c *command) kafkaStart(cmd *cobra.Command, args []string) error {
 
 	log.CliLogger.Tracef("Pull confluent-local image success")
 
-	if err := c.prepareAndSaveLocalPorts(); err != nil {
-		return err
+	if c.Config.IsTest {
+		if err := c.prepareAndSaveLocalPortsForTest(); err != nil {
+			return err
+		}
+	} else {
+		if err := c.prepareAndSaveLocalPorts(); err != nil {
```

Comment:
Can we pass `isTest` into this function? Looks like the implementation of these two functions is very similar and could be combined.